### ISSUE TITLE
Not blocking ads

### ISSUE THREAD

**Issue by carlos12silva94** [abba23/spotify-adblock#65]
I had the problem where spotify show the offline error. Then i update the adblock version, but now it doesn't block the ads.
Ubuntu 18.04 LTS

**Comment by enovella** [comment]
Working flawlessly on Ubuntu 18.04 LTS. Please read the documentation and explain further your issues.

**Comment by carlos12silva94** [comment]
I did this:
$ git clone https://github.com/abba23/spotify-adblock.git
$ cd spotify-adblock
$ make
$ sudo make install
$ LD_PRELOAD=/usr/local/lib/spotify-adblock.so spotify

once in a while, the ad starts to play. What else is supposed to do?

**Comment by enovella** [comment]
Can we see the log of the last command?

**Comment by carlos12silva94** [comment]
This is right after i run it:
![image](https://user-images.githubusercontent.com/26636699/146413293-993f3279-c3a4-4b3d-b868-a8e5ee326098.png)
When i get an ad it turns into:
![image](https://user-images.githubusercontent.com/26636699/146414811-0bc749fb-3fd0-4c16-ac9c-2080f91212e2.png)

**Comment by abba23** [comment]
That log looks like it should be working. Are you perhaps using a very outdated version of Spotify? The latest version should be `1.1.72.439.gc253025e`.

**Comment by enovella** [comment]
This is working to my eyes but you got an ERROR in the first line. Have you searched for that?  Please use text instead of pictures, now we cannot copy the ERROR to help you further.

**Comment by BwntyHntr** [comment]
I'm experiencing the same issue on arch Linux

I've noticed that abba23 modified the config file which has allow list and deny list to temp fix the issue.

From what I have experienced, Spotify linked an ad domain to their main infrastructure which means by blocking it, you cannot select any song to play as it shows "content not available".

The good news that it doesn't show any ads while playing music, only banner ads.
Perhaps something can be one about it in the future.
